### PR TITLE
Fix/search list many orgs modal

### PR DESCRIPTION
**Pull request by Josmar-jr**
#### What problem is this solving?

Before, it was not possible to filter the organization according to the value entered in the input at change organization modal

#### How to test it?

1. Navigate to [Workspace](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-organizations/organizations#/organizations)
2. Select the settings option in tabs
3. Enable "Usar modal para trocar de organização"
4. Navigate to [Workspace](https://josmarjr--b2bstore005.myvtex.com/)
5. Do the login
6. Click on the button "Mudar organização"
7. Dispatch a new search at input

[Workspace](https://josmarjr--b2bstore005.myvtex.com/)

#### Screenshots or example usage:
Screenshot 2024-07-03 at 10 24 01
Screenshot 2024-07-03 at 10 23 46

### CODE REVIEW

File: react/components/UserWidget.tsx [hunk 1]
```diff
@@ -463,6 +442,41 @@ const UserWidget: VtexFunctionComponent<UserWidgetProps> = ({
...
+          .slice(0, 15)
+          ?.sort(sortOrganizations) ?? []
+    }
+
+    setOrganizationsState({
+      ...organizationsState,
+      dataList,
+      totalDataList: dataList?.length,
+    })
+  }
```
Comment (ArthurTriis1) [inline react/components/UserWidget.tsx:478]:
```suggestion
  const handleSearchOrganizations = (e: any) => {
    const { value }: { value: string } = e.target
    let dataList

    setSearchTerm(e.target.value)
    
    dataList = userWidgetData?.getOrganizationsByEmail?.sort(
        sortOrganizations
      )

    if (value.trim() != '') {
        dataList =
        userWidgetData?.getOrganizationsByEmail
          ?.filter((organization: any) => {
            return (
              organization.organizationName
                .toLowerCase()
                .includes(value.toLowerCase()) ||
              organization.costCenterName
                .toLowerCase()
                .includes(value.toLowerCase())
            )
          })
          ?.slice(0, 15) ?? []
    }

    setOrganizationsState({
      ...organizationsState,
      dataList,
      totalDataList: dataList?.length,
    })
  }
```

**Review by ArthurTriis1 — COMMENTED** [review]
I tried to reproduce the issue and enabled the modal in the admin panel.

Screenshot 2024-07-09 at 13 24 13

However, the modal button does not appear.

Screenshot 2024-07-09 at 13 23 02

Is there anything else I need to configure?

**Comment by Josmar-jr** [comment]
@ArthurTriis1 just enabling the modal should work

**Comment by ArthurTriis1** [comment]
> @ArthurTriis1 just enabling the modal should work

Ok, I will try again, but the current workspace is broken, can you fix it?
Screenshot 2024-07-12 at 09 49 22

**Comment by Josmar-jr** [comment]
@ArthurTriis1 Here it is working normally. Please check if the project branch is up to date
Screenshot 2024-07-12 at 10 09 47
Screenshot 2024-07-12 at 10 11 04

**Comment by ArthurTriis1** [comment]
> @ArthurTriis1 Here it is working normally. Please check if the project branch is up to date 

It is working now! And I could test the modal, it is working fine!

**Review by ArthurTriis1 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,44 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Adjust search list many organations at change organization modal
 
 ## [1.31.3] - 2024-04-22
 
 ### Fixed
 
-- Save add, edit and delete address within the Cost Center in admin
-
-### Fixed
-
```
Comment (ArthurTriis1) [inline CHANGELOG.md:47]: Some changes are being removed from the changelog, which is causing the checks to break. Could you please add those changes back and update the changelog?

**Review by ArthurTriis1 — CHANGES_REQUESTED** [review]

**Comment by Josmar-jr** [comment]
I took the CHANGELOG from the master branch and added the feat from this PR, right?
@ArthurTriis1

**Review by ArthurTriis1 — APPROVED** [review]

**Pull request merged**